packages/ui/app/src/api-page/endpoints/EndpointContent.tsx

```diff
@@ -200,11 +217,11 @@ export const EndpointContent = React.memo<EndpointContent.Props>(function Endpoi
                             marginTop: titleHeight - 40,
                         }}
                     >
-                        <EndpointExamples endpoint={endpoint} />
+                        {example && <EndpointExample endpoint={endpoint} example={example} />}
```

Comment:
thought eslint would catch this but i guess not - for clarity I always do `!= null` so it's clear what we're checking for, since loads of other things are falsy

Reply:
oh right, old habits... will change. i use `&&` only when I know that the value can be either an object or nulllish (e.g. when `""`, `0` etc. are not a possibility)